tools/pd-ctl/pdctl/command/scheduler.go

```diff
@@ -54,29 +54,45 @@ func NewSchedulerCommand() *cobra.Command {
 // NewPauseSchedulerCommand returns a command to pause a scheduler.
 func NewPauseSchedulerCommand() *cobra.Command {
 	c := &cobra.Command{
-		Use:   "pause <scheduler> <delay>",
+		Use:   "pause <scheduler> <delay_second>",
 		Short: "pause a scheduler",
-		Run:   pauseOrResumeSchedulerCommandFunc,
+		Run:   pauseSchedulerCommandFunc,
 	}
 	return c
 }
 
-func pauseOrResumeSchedulerCommandFunc(cmd *cobra.Command, args []string) {
-	if len(args) != 2 && len(args) != 1 {
+func pauseSchedulerCommandFunc(cmd *cobra.Command, args []string) {
+	if len(args) != 2 {
+		cmd.Usage()
+		return
+	}
+	delay, err := strconv.Atoi(args[1])
```

Comment:
the delay is int but the api suport int64. 
`
if err := h.PauseOrResumeScheduler(name, int64(t)); err != nil {
		h.r.JSON(w, http.StatusInternalServerError, err.Error())
		return
	}
`